WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -362,13 +378,13 @@ private extension PaymentGatewayAccount {
     }
 }
 
-private extension CardPresentPaymentsPlugins {
+private extension CardPresentPaymentsOnboardingUseCase {
     // This was moved from Yosemite so it can check the feature flag
-    // In a future iteration, we might want to store all the configuration in the WooCommerce layer
-    var supportedCountryCodes: [String] {
-        switch self {
+    // In a future iteration, we might want to store all the configuration in a better place
```

Comment:
Github's highlighting here makes me think the comment is about the configuration going to heaven 😅